### ISSUE TITLE
Action & State always expanded

### ISSUE THREAD

**Issue by m2mathew** [gaearon/redux-devtools-log-monitor#58]
We are implementing the fantastic Redux-Dev-Tools in our app. It promises to help us understand things much better.

I think we have this set up correctly to collapse the action and state in the `LogMonitor`. Our state tree is rather large with 158 keys...having it expanded and toggling dialogs and having request/success httpPuts, etc. is makes having the state expanded hard to deal with.

Here is our implementation of DockMonitor and LogMonitor:
```javascript
// Node modules
import DockMonitor from 'redux-devtools-dock-monitor';
import LogMonitor from 'redux-devtools-log-monitor';
import React from 'react';
import { createDevTools } from 'redux-devtools';


// Component definition
export default createDevTools(
  <DockMonitor
    changePositionKey="ctrl-q"
    defaultIsVisible={false}
    defaultPosition="right"
    defaultSize={0.3}
    fluid
    toggleVisibilityKey="ctrl-h"
  >
    <LogMonitor
      expandActionRoot={false}
      expandStateRoot={false}
      theme="monokai"
    />
  </DockMonitor>
);
```

We are using these package versions for this fun stuff:

```javascript
"redux-devtools": "3.x",
"redux-devtools-dock-monitor": "1.x",
"redux-devtools-log-monitor": "1.x"
```

Thanks for any pointers!

**Comment by m2mathew** [comment]
I have spent a little more time looking into this without much success. The `src` code for this all looks like the `expandActionRoot` and `expandStateRoot` props are passing down as I would expect.

The only thing I notice is that the package.json for this repo has this package installed
`"react-json-tree": "^0.7.0"`

but my local project has this package installed
`"react-json-tree": "^0.6.6"`

Would that cause the difference in the way that the nodes respond to the "expand" props?

**Comment by zalmoxisus** [comment]
The different version of `react-json-tree` shouldn't be an issue. It is not published because of [some optimizations concerns there](https://github.com/gaearon/redux-devtools-log-monitor/pull/54#discussion_r63805314).

`expandActionRoot/expandStateRoot` props was [introduced in the earlier versions](https://github.com/gaearon/redux-devtools-log-monitor/pull/16). However, in the older versions of `react-json-tree`, the support for `expandRoot` could be broken, didn't investigate it.

If you find the problem, please submit a pr here or to `react-json-tree`.

**Comment by m2mathew** [comment]
@zalmoxisus The issue is with `react-json-tree` package being outdated. I will make a PR to address this. Thanks!

**Comment by zalmoxisus** [comment]
Fixed in `v1.1.0`. Now by default everything is collapsed ([see the release notes](https://github.com/gaearon/redux-devtools-log-monitor/releases/tag/v1.1.0)).
